src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/PropertyInfoPassTest.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\Compiler;
 
+use Prophecy\Argument;
```

Comment:
> Btw I've used prophecy here because the test is more accurate than with old mocks. I test that all methods of Definition and ContainerBuilder that must be called are really called. I don't know if it's even possible to that using old PHPUnit mocks.

Isn't this what `expects($this->once())` does in phpunit mocks?

Although I love prophecy I think we should be consistent for the same reasons @WouterJ mentioned.